VAMobile/src/screens/HomeScreen/ProfileScreen/PersonalInformationScreen/GenderIdentityScreen/GenderIdentityScreen.tsx

```diff
@@ -70,9 +78,19 @@ const GenderIdentityScreen: FC<GenderIdentityScreenProps> = ({ navigation }) =>
     setGenderIdentity(value)
   }
 
+  const updateGenderIdentity = () => {
+    if (genderIdentity) {
```

Comment:
Sorry, I merged too quickly before I saw this 😅 I added this check inside the function because the mutation function complains that `genderIdentity` can't be of type `string | undefined`, since it expects the parameter to be just a string. I'll probably update the one from the `onSave` function and just have it return early after setting the error if `genderIdentity` is undefined. Could also remove the logic in this function directly into the `onSave` function. The main reason why I have this logic wrapped in a function is because of the retry function that gets passed to the snack bar on errors